int32 OFF = 0
int32 ON = 1
int32 FAST_BLINK = 2
int32 SLOW_BLINK = 3
std_msgs/Int32 type